README.md

```diff
@@ -68,7 +68,7 @@ N_DEVICES=8
 torchrun --nproc_per_node=${N_DEVICES} train_ir.py config_infonce.yaml
 ```
 
-Data file is a datasets.Dataset, each sample is an object like this:
```

Comment:
the datasets here is package name https://huggingface.co/docs/datasets/index
I think we don't need to do change here

Reply:
Sure, I had some misunderstanding. Maybe we should use `datasets.Dataset` instead.